src/vs/workbench/parts/output/browser/output.contribution.ts

```diff
@@ -45,3 +29,25 @@ registerMode({
 	moduleId: 'vs/workbench/parts/output/common/outputMode',
 	ctorName: 'OutputMode'
 });
+
+// Register Output Panel
+(<panel.PanelRegistry>platform.Registry.as(panel.Extensions.Panels)).registerPanel(new panel.PanelDescriptor(
```

Comment:
@isidorn do not forget to add the panel to the same buildfile.js where we added REPL so that the loader can load it